apps/rush/src/actions/RushCommandLineParser.ts

```diff
@@ -90,6 +92,15 @@ export default class RushCommandLineParser extends CommandLineParser {
     }
   }
 
+  private _ensureEnvironment(): void {
+    /* tslint:disable-next-line:no-string-literal */
+    let environmentPath: string = process.env['PATH'];
+    environmentPath = path.join(this.rushConfiguration.commonTempFolder, 'node_modules', '.bin') +
```

Comment:
But what if it's the wrong version? The expected tool won't be used.

---
In reply to: [119753539](https://github.com/Microsoft/web-build-tools/pull/218#discussion_r119753539) [](ancestors = 119753539,119751422)